source/_components/climate.toon.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Toon van Eneco"
```

Comment:
Title differs from rest of document. Would suggest only Toon since the type is view-able on the component page.